Content.Client/GameObjects/Components/Body/Surgery/SurgeryWindow.cs

```diff
@@ -105,7 +105,7 @@ public SurgeryButton(int callbackData)
                     (DisplayText = new Label
                     {
                         VerticalAlignment = VAlignment.Center,
-                        Text = "N/A",
+                        Text = Loc.GetString("generic-not-available-shorthand"),
```

Comment:
I'm not sure our stance on having generic entries like this.

Given that there's no context to this particular usage it's *probably* fine, but just making a note here for yourself/other reviewers to consider.

Reply:
I tried to only use generic tokens for super simple stuff, like Yes/No, or stuff which was repeating a lot in different places, like "You have no hands".

Reply:
As far as I've seen you've done a good job of keeping them context-independent, so long as that's true I think generic entries are fine.